test/karma.conf.js

```diff
@@ -0,0 +1,50 @@
+var path = require('path');
+
+module.exports = function(config) {
+  config.set({
+    basePath: '../',
+    browsers: ['PhantomJS'],
+    coverageReporter: {
+      type: 'lcov',
+      dir: 'test/coverage/'
+    },
+    files: [
+      'test/bind-polyfill.js',
+      'packages/*/test/**/*.js',
+    ],
+    frameworks: ['mocha', 'chai', 'sinon-chai'],
+    reporters: ['mocha', 'coverage'],
+    preprocessors: {
+      'packages/**/*.js': ['webpack']
+    },
+    webpack: {
+      resolve: {
+        extensions: ['', '.js', '.jsx'],
+        alias: {
+          helpers: path.join(__dirname, 'helpers'),
+          fixtures: path.join(__dirname, 'fixtures'),
+          'component-playground': path.join(__dirname, '../packages/react-component-playground/src')
+        }
+      },
+      module: {
+        loaders: [{
+          test: /\.jsx?$/,
+          exclude: /node_modules/,
+          loader: 'babel',
```

Comment:
Pretty cool. Not sure if this repo would benefit, tho, since the test cases are pretty verbose and it doesn't have the crazy nesting you guys have (not to mention the repetitiveness between similar components).

![image](https://cloud.githubusercontent.com/assets/250750/17945777/fb92d902-6a4d-11e6-855e-0a7396701cc5.png)